### PR TITLE
Homepage for new release

### PR DESCRIPTION
**Pull request by maccxs**
More compact description of emoji states
Description of gameplay
Custom color for cards
Text adapated  for join us

### CODE REVIEW

**Comment by holke** [comment]
Finde ich gut :)
Muss es mir noch in der Homepage Version anschauen (reviewe gerade von Handy aus)

File: index.html [hunk 1]
```diff
@@ -174,33 +177,53 @@ <h5 class="card-title"> Erholter<br> Spieler</h5>
...
+            <li>Lenke den Spieler durch die Benutzung der Pfeiltasten auf der Tastatur.</li>
+            <li>Q-Taste beendet das Level.</li>
+            <li>Esc-Taste beendet den Vollbildmodus.</li>
+             <li><b>Sandbox</b>
+                <ul>
+                    <li>Dein Ziel ist es, Dich selbst nicht zu infizieren. Und wenn es Dich dann doch getroffen hat, solltest Du möglichst wenig andere Emojis anstecken.</li>
+                     <li>   In unserer Simulation halten sich alle Mitbürger mit verschiedener Wahrscheinlichkeit an das vorgegebene Social-Distancing.
+                            Sich zu nahe zu kommen wird aber immer weiter vermieden, je höher Du diesen Faktor setzt. 
+                            Das kann glücklicherweise auch dazu führen, dass sich niemand mehr ansteckt.</li>
+                     <li>   Zusätzlich kannst Du mir der Anzahl der Emojis experimentieren. </li>
```
Comment (holke) [inline index.html]:
```suggestion
                     <li>   Zusätzlich kannst Du mit der Anzahl der Emojis experimentieren. </li>
```

**Comment by holke** [comment]
Can we move the smileys down a little bit?

grafik

They touch the top of their box. I think it would look better if they did not.

Also, can we align them all at their bottom? The player with hat sticks out and looks odd.
I think this can be solved if all pics are at the same bottom line, not the same top line as is currently.

**Comment by maccxs** [comment]
> I think this can be solved if all pics are at the same bottom line, not the same top line as is currently.

If this were word and not bootstrap it would be this easy :-)

**Pull request merged**